gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NexusPageHeaderExample from './NexusPageHeaderExample';
+
+const nexusPageHeaderExampleCode = require('!!raw-loader!./NexusPageHeaderExample').default;
+
+const NxNexusPageHeaderPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxNexusPageHeader</code> is a React component encapsulating the Sonatype Nexus
+        branded page header logo and structure.
```

Comment:
Do we want to elaborate on when this should be used vs the other header?

Reply:
> Do we want to elaborate on when this should be used vs the other header?

How's this? https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/96b2b2be0323fa7840f525fef295a3f1284c4ba7